ask.py

```diff
@@ -0,0 +1,25 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
this part works on my machine, and I'm able to get the environment variables stored in `.env` file

Reply:
So now we are sure that the problem is on my dotenv library, I'll see how I could fix it